R/runSSMSE.R

```diff
@@ -522,6 +522,19 @@ run_SSMSE_iter <- function(out_dir = NULL,
   assertive.types::assert_is_any_of(nscen, c("integer", "numeric"))
   if (!is.null(sample_struct)) {
     assertive.types::assert_is_list(sample_struct)
+    ## Added lines below to account for different naming conventions  
```

Comment:
I'm not sure why this is happening. It looks like these are the r4ss names, which should get [converted later](https://github.com/nmfs-fish-tools/SSMSE/blob/93ff768b6e8ebe998cf24afbaa050b1d5e7c69bb/R/runSSMSE.R#L588) in this function. Are the wrong column names coming out of `create_sample_struct()`? Perhaps you could share your "line by line" code (or at least describe the process more)? I'm not sure I understand how running it is different than running `run_SSMSE()`

Reply:
Good question -- this one is taking me a bit longer to figure out. I think I have an idea of what happened. 

When running the code line-by-line, I must have run into errors below the "Convert sample_struct names" section, where my original sample_struct object was written over with new labels. My new sample_struct object would then be labeled with the corrected labels (e.g., year, seas, fleet, catch_se, etc.), not the original labels (e.g., Yr, Seas, FltSvy, etc). Then, to try to find and correct the error, I would re-run the code without re-creating the original sample_struct object. When I went back to re-run the code with the new sample_struct labels, the check_sample_struct() command ([Line 538](https://github.com/nmfs-fish-tools/SSMSE/blob/93ff768b6e8ebe998cf24afbaa050b1d5e7c69bb/R/runSSMSE.R#L538)) doesn't work with the new/corrected labels-- it only works with the original labels.  

So, I think you're right that we can disregard this change. 

I'll have to add in some sort of virgin sample_struct object (sample_struct0) into my code when I'm running line-by-line so that I can retrieve the original sample_struct after writing over it in the "Convert sample_struct names" section. 

Hope this was clear enough!

Reply:
Thanks! I'll check out the browser() function. 

Upon further exploration of my notes -- it looked like I was running into the issue while running the loop line-by-line. After re-writing sample_struct() in the first iteration (i=1) it was not working in the second iteration (i=2). Using this insight to reset my sample_struct = sample_struct0 in line 524 in my line-by-line script. Just documenting here for posterity!